src/shared/ui/survey/CheckBox/types.ts

```diff
@@ -0,0 +1,40 @@
+export type Item = {
+  name: { en: string };
+  value: number;
+  color: string;
+  isVis: boolean;
+  description: string;
+  image?: string;
+};
+//
+// type RefactoredConfig = {
```

Comment:
Please comment above the commented code why did you keep the commented code here?
Better to avoid it if we don't have strong reasons.